staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -1803,6 +1807,18 @@ func (c *Cloud) GetZoneByNodeName(ctx context.Context, nodeName types.NodeName)
 
 }
 
+func isAWSErrorInstanceNotFound(err error) bool {
+	if err != nil {
+		if awsError, ok := err.(awserr.Error); ok {
+			if awsError.Code() == "InvalidInstanceID.NotFound" {
```

Comment:
Is there a const for the string `InvalidInstanceID.NotFound` in the AWS SDK?

Reply:
there is one called: UnsuccessfulInstanceCreditSpecificationErrorCodeInvalidInstanceIdNotFound

https://raw.githubusercontent.com/aws/aws-sdk-go/v1.28.11/service/ec2/api.go

Reply:
should we use that?

Reply:
ok